charts/memgraph/README.md

```diff
@@ -32,6 +32,7 @@ The following table lists the configurable parameters of the Memgraph chart and
 | `replicaCount`                              | Number of Memgraph instances to run. Note: no replication or HA support.                            | `1`                                     |
 | `affinity.nodeKey`                          | Key for node affinity (Preferred)                                                                   | `""`                                    |
 | `affinity.nodeValue`                        | Value for node affinity (Preferred)                                                                 | `""`                                    |
+| `nodeSelector`                              | Constrain which nodes your Memgraph pod is eligible to be scheduled on, based on the labels on the nodes. Left empty by default.                       | `{}`                   |
```

Comment:
Should this change be here?

Reply:
I forgot to update this [here](https://github.com/memgraph/helm-charts/pull/39), so adding it here, since it is related to the review: https://github.com/memgraph/documentation/pull/960